src/utils/environment.ts

```diff
@@ -7,22 +7,32 @@
 import Constants from 'expo-constants';
 
 const ENV = {
+  local: {
+    apiUrl: 'http://097ab81b.ngrok.io',
+    auth0ClientId: '0ngrMLtiiqOeY7ADbMSOq71tYb50LiUc',
+    auth0Domain: 'https://matterhorn-prototype.auth0.com',
+    enableAuth: false,
+    message: 'Hello, local!'
+  },
   dev: {
-    apiUrl: 'http://matterhornapiservice-env.eba-qjezc5kq.us-east-1.elasticbeanstalk.com',
+    apiUrl:
+      'http://matterhornapiservice-env.eba-qjezc5kq.us-east-1.elasticbeanstalk.com',
```

Comment:
I think this should probably be env-dev.

Reply:
That's right, though I never changed this, so it's always been targeting prod :(
I even looked at master, and it's wrong there, too. Updating now.